apps/firestorm_web/test/unit/earmark/auto_linker_test.exs

```diff
@@ -7,4 +7,10 @@ defmodule FirestormWeb.Earmark.AutoLinkerTest do
     expected = "<a href=\"http://slashdot.org\">http://slashdot.org</a><br/>"
     assert AutoLinker.run(body) == expected
   end
+
+  test "does nothing to a naked URL wrapped in quotes" do
+    body = "\"http://slashdot.org\""
```

Comment:
That seems like a hack to get around the problem; it might be good enough for now but I'm not sure we'll want this long-term.